ZupIT/ritchie-cli
# Formula inputs by flags

**- What I did**
I allow the formula inputs by flags:
![image](https://user-images.githubusercontent.com/17505818/96628709-f9f18600-12e8-11eb-983a-e1c5f2f70623.png)

- When an input by the flag is empty the Ritchie calls the prompt to the user to insert this value.
- Edited the code owners 
- Created an input resolver for formula runner
- Split the inputs.go into `input/prompt, input/stdin, and input/flag` to make easier the maintainability.
- improved how to add reserved flags
- now you can use the input by stdin without the `--stdin` flag for formulas command.

**- How to verify it**
1 - Checkout in this branch
2 - Build Ritchie locally `make build-{your-operational system}` and replace you locally installation
3 - Show your formula flags with `rit formula name --help`, you can see the inputs name such as flags
4 - Run your formulas with flags

**- Description for the changelog**
Allow formula inputs with flags
